examples/bert/src/bert_layers.py

```diff
@@ -815,13 +815,12 @@ def forward(
         if (input_ids is not None) == (inputs_embeds is not None):
             raise ValueError('Must specify either input_ids or input_embeds!')
 
-        masked_lm_labels = labels
-        if masked_lm_labels is None:
-            raise ValueError('Mosaic BertForMaskedLM requires a labels tensor.')
-
-        return_dict = return_dict if return_dict is not None else self.config.return_dict
+        if labels is None:
+            masked_tokens_mask = None
+        else:
+            masked_tokens_mask = labels > 0
 
-        masked_tokens_mask = masked_lm_labels > 0
+        return_dict = return_dict if return_dict is not None else self.config.use_return_dict
```

Comment:
does this work? I've been seeing this syntax used by others and I feel like it helps reduce line length, haha
```suggestion
        return_dict = return_dict or self.config.use_return_dict
```

Reply:
^I don't think this works because 
```
In [1]: False or True
Out[1]: True
```

Reply:
Oooh, good catch @dakinggg. I was also under the impression that that `a or b` was a special syntax for `a if a is not None else b`. Basically, you have to keep the long version because `return_dict` has a type signature of `Optional[bool]`. Today I learned...

Reply:
haha I try to avoid Falseyness in python as much as possible. So many subtle bugs